packages/form-generator/src/local_codegen_graphql_form_generator.ts

```diff
@@ -98,20 +98,23 @@ export class LocalGraphqlFormGenerator implements GraphqlFormGenerator {
       renderTypeDeclarations: true,
       apiConfiguration: {
         dataApi: 'GraphQL',
-        fragmentsFilePath: path.join(
+        fragmentsFilePath: `./${path.join(
```

Comment:
Does this work on windows?

Reply:
This string is generated straight into the UI Form. If it is left out, the string is sent as `graphql/fragments`, which node attempts to resolve as a node module. I can get a windows vm set up to verify this.

Reply:
Should we get some integration(better)/e2e(worse) test for this where we attempt to use generated code?

Reply:
This is definitely a missing test vector. I'll get a test written for this.

Reply:
https://github.com/aws-amplify/amplify-backend/pull/650/files#r1393530504